cou/utils/app_utils.py

```diff
@@ -75,6 +75,30 @@ async def get_instance_count(unit: str, model: COUModel) -> int:
     )
 
 
+async def enable_nova_compute_scheduler(unit: str, model: COUModel) -> None:
+    """Enable nova-compute scheduler, so the unit can create new VMs.
+
+    :param unit: Name of the nova-compute unit where the action runs on.
+    :type unit: str
+    :param model: COUModel object
+    :type model: COUModel
+    """
+    action_name = "enable"
+    await model.run_action(unit_name=unit, action_name=action_name)
+
+
+async def disable_nova_compute_scheduler(unit: str, model: COUModel) -> None:
+    """Disable nova-compute scheduler, so the unit cannot create new VMs.
+
+    :param unit: Name of the nova-compute unit where the action runs on.
+    :type unit: str
+    :param model: COUModel object
+    :type model: COUModel
+    """
+    action_name = "disable"
+    await model.run_action(unit_name=unit, action_name=action_name)
```

Comment:
I don't understand why we can't add `model.run_action` as coro into an upgrade step of nova-compute application class (similar to how you are handling `pause` and `resume` to base application class). Any specific reason that I'm missing?

Reply:
I see your point. I was kind following the implementation of instance-count, but giving a second thought those actions probably won't be called outside the Nova-Compute class as it happens with instance-count.

I think it's ok leaving this way until the next PR for the nova-compute implementation. If I confirm this, I can move it to the Nova-Compute class. What do you think?

Reply:
I agree, these functions are useless as utils, since they will be used only in the NovaCompute class.

Reply:
Removed from this PR. It's a simple function that can be introduced with NovaCompute class.